splice_si_api/src/main/java/com/splicemachine/si/impl/SimpleTxnFilter.java

```diff
@@ -189,6 +190,13 @@ protected void doResolve(DataCell data,long ts){
 		 * it matches, then we can see it.
 		 */
         long timestamp=data.version();//dataStore.getOpFactory().getTimestamp(data);
+        SIDriver driver = SIDriver.driver();
+        IgnoreTxnSupplier ignoreTxnSupplier = null;
+        if (driver != null) {
```

Comment:
This looks weird, when could the driver be null? I don't think this should be allowed.